recipes-bsp/xt-rpi-u-boot-scr/files/boot.cmd.xen.in

```diff
@@ -0,0 +1,31 @@
+fatload @@BOOT_MEDIA@@ 0 @@XEN_DTB_ADDR@@ @@XEN_DTB@@
+fdt addr @@XEN_DTB_ADDR@@
+fdt resize 2048
+fdt chosen
+
+fatload @@BOOT_MEDIA@@ 0 @@XEN_IMG_ADDR@@ @@XEN_IMAGE@@
+fdt set /chosen/modules/module@1 reg <0x0 @@XEN_IMG_ADDR@@ 0x0 0x${filesize} >
+
+fatload @@BOOT_MEDIA@@ 0 @@XENPOLICY_IMG_ADDR@@ @@XENPOLICY_IMAGE@@
+fdt set /chosen/modules/module@2 reg <0x0 @@XENPOLICY_IMG_ADDR@@ 0x0 0x${filesize} >
+
+fatload @@BOOT_MEDIA@@ 0 @@DOM0_IMG_ADDR@@ @@DOM0_IMAGE@@
+fdt set /chosen/dom0 reg <0x0 @@DOM0_IMG_ADDR@@ 0x0 0x${filesize} >
+
+fdt mknod /chosen/domU0 module@@@DOMU0_IMG_ADDR@@
+fdt set /chosen/domU0/module@@@DOMU0_IMG_ADDR@@ compatible  "multiboot,kernel" "multiboot,module"
+fdt set /chosen/domU0/module@@@DOMU0_IMG_ADDR@@ bootargs "console=ttyAMA0 earlycon=xen earlyprintk=xen clk_ignore_unused root=/dev/mmcblk0p2 rootfstype=ext4 rootwait"
+fatload @@BOOT_MEDIA@@ 0 @@DOMU0_IMG_ADDR@@ @@DOMU0_IMAGE@@
+fdt set /chosen/domU0/module@@@DOMU0_IMG_ADDR@@ reg <0x0 @@DOMU0_IMG_ADDR@@ 0x0 0x${filesize} >
+
+fdt mknod /chosen/domU0 module@@@DOMU0_DTB_ADDR@@
+fdt set /chosen/domU0/module@@@DOMU0_DTB_ADDR@@ compatible  "multiboot,device-tree" "multiboot,module"
+fatload @@BOOT_MEDIA@@ 0 @@DOMU0_DTB_ADDR@@ @@DOMU0_DTB@@
+fdt set /chosen/domU0/module@@@DOMU0_DTB_ADDR@@ reg <0x0 @@DOMU0_DTB_ADDR@@ 0x0 0x${filesize} >
+
+setenv fdt_high 0xffffffffffffffff
+
+fdt print /chosen
+printenv
```

Comment:
please no prints

Reply:
Ok

Reply:
Prints must stay for now! It's very important for current dev stage